plugins/topology/src/vm-models.ts

```diff
@@ -5,10 +5,27 @@ export const VirtualMachineGVK: GroupVersionKind = {
   apiGroup: 'kubevirt.io',
   kind: 'VirtualMachine',
 };
+
+export const VirtualMachineInstanceGVK: GroupVersionKind = {
+  apiVersion: 'v1',
+  apiGroup: 'kubevirt.io',
+  kind: 'VirtualMachineInstance',
+};
+// export const ReplicationControllerGVK: GroupVersionKind = {
+//   apiVersion: 'v1',
+//   kind: 'ReplicationController',
+// };
```

Comment:
We don't show RepilicaControllers at the moment?

Reply:
We don't show RepilicaControllers